using-timescaledb/continuous-aggregates.md

```diff
@@ -151,6 +151,25 @@ will be spawned immediately following completion. This means that the part of th
 data processed by previous jobs will be available even before the
 materialization is fully caught up.
 
+The `timescaledb.ignore_invalidation_older_than` parameter controls
+how modifications (inserts, updates, and deletes) will trigger update
+of the continuous aggregate. If modifications are done to the
+underlying table (or tables), it invalidates the already computed
+aggregate and the aggregate has to be recomputed.
+
+By default, all modifications trigger a recompute of the continuous
+aggregate. If the `ignore_invalidation_older_than` parameter is set to
+a duration, modifications with a timestamp that is older will be
+ignored and not trigger a recompute of the continuous aggregate.
+
+As a result, refresh jobs will complete faster since they are only
+dealing with new data as part of the refresh.
+
+If setting `ignore_invalidation_older_than` you need to consider if
```

Comment:
Data retention policies for hyperatbles are affected by continuous aggregates. Say you have a raw table and a cont agg defned on it. If the hypertable has drop chunks wants to drop chunks older than 1 month, then the ignore_invalidation-older_than parameter for cont. aggregates that are dependent on it should be 1 month. So dropping the raw data does not affect the continuous aggregate. (I think the detailed explanation can wait for the data retention policy PR. But adding a link here to data retention doc is probably useful).

Reply:
This sounds dangerous, should we not disallow dropping chunks if it can affect a continuous aggregate? Need to have a good error message as well.

Reply:
We check if there are cont. aggregates dependent on the chunk before it can be dropped. Errors, otherwise.

Reply:
Maybe that should be mentioned in the documentation?

Reply:
I'll add this to the data retention PR.